package-lock.json

```diff
@@ -1157,6 +1108,18 @@
             "mime": "^2.2.0"
           }
         },
+        "logger-sharelatex": {
+          "version": "2.1.1",
+          "resolved": "https://registry.npmjs.org/logger-sharelatex/-/logger-sharelatex-2.1.1.tgz",
+          "integrity": "sha512-qqSrBqUgHWnStxtTZ/fSsqPxj9Ju9onok7Vfm3bv5MS702jH+hRsCSA9oXOMvOLcWJrZFnhCZaLGeOvXToUaxw==",
+          "requires": {
+            "@google-cloud/logging-bunyan": "^3.0.0",
+            "@overleaf/o-error": "^3.0.0",
+            "bunyan": "^1.8.14",
+            "raven": "^2.6.4",
+            "yn": "^4.0.0"
+          }
+        },
```

Comment:
Somehow logger-sharelatex-2.1.1 is still appearing in package-lock.json.

Reply:
It's under `@overleaf/object-persistor`